src/interfaces/Config.ts

```diff
@@ -13,16 +13,49 @@ export interface IConfigGeography {
   node: IConfigGeographyNode
   children: IConfigGeography[]
 }
+interface IChakraSelect extends OptionBase {
+  value: string
+  label: string
+}
+
+export interface IConfigLanguageSorted extends IChakraSelect {}
+
+export interface IConfigCorpus extends IChakraSelect {}
+
+export interface IConfigLanguageSorted extends OptionBase {
+  value: string
+  label: string
+}
 
 interface IConfigMeta {
   allow_any?: boolean
   allow_blanks: boolean
   allowed_values: string[]
 }
 
-export interface IConfigLanguageSorted extends OptionBase {
-  value: string
-  label: string
+export interface IConfigTaxonomyCCLW {
```

Comment:
Wondering how we are going to manage this when we have many more corpora :thinking:

Reply:
You and me both 🤔